src/graphql/query/shared/hasEverLoggedIn.graphql

```diff
@@ -1,3 +1,13 @@
 query hasEverLoggedIn {
 	hasEverLoggedIn @client
+	my {
```

Comment:
The hasEverLoggedIn query is used in other components, so let's put this in a new query and add it as another call in the prefetch:
```
client.query({ query: hasEverLoggedInQuery }),
client.query({ query: optimizelyUserDataQuery }),
fetchAllExpSettings(config, client, {
	query: route?.query,
	path: route?.path
})
```